handlers/main.yml

```diff
@@ -17,7 +17,7 @@
   become_user: "{{ atom_user }}"
   environment:
     - "{{ atom_pool_php_envs }}"
-    - PATH: "{{ ansible_env.PATH }}:/opt/rh/rh-php{{ php_version }}/root/bin/"
+    - PATH: "{{ ansible_env.PATH }}:{{ php_rh_centos_path | default('/opt/rh/rh-php{{ php_version }}/root/bin')}}"
```

Comment:
I gather the / after "/root/bin" was redundant (removal intentional)? Seems consistent with _handlers/main.yml_ that way, if unchanged in _tasks/main.yml_ (context/usage)?

Reply:
Assuming as I am line will work, this question resolved.